hive/src/main/java/org/apache/zeppelin/hive/HiveInterpreter.java

```diff
@@ -0,0 +1,203 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.zeppelin.hive;
+
+import java.sql.*;
+import java.util.List;
+import java.util.Properties;
+
+import com.nflabs.zeppelin.interpreter.*;
+import org.apache.commons.lang.StringUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.nflabs.zeppelin.interpreter.InterpreterResult.Code;
+import com.nflabs.zeppelin.scheduler.Scheduler;
+import com.nflabs.zeppelin.scheduler.SchedulerFactory;
+
+/**
+ * Hive interpreter for Zeppelin.
+ */
+public class HiveInterpreter extends Interpreter {
+  Logger logger = LoggerFactory.getLogger(HiveInterpreter.class);
+  int commandTimeOut = 600000;
+
+  static final String HIVESERVER_URL = "hive.hiveserver2.url";
+  static final String HIVESERVER_USER = "hive.hiveserver2.user";
+  static final String HIVESERVER_PASSWORD = "hive.hiveserver2.password";
+
+  static {
+    Interpreter.register(
+      "hive",
+      "hive",
+      HiveInterpreter.class.getName(),
+      new InterpreterPropertyBuilder()
+        .add(HIVESERVER_URL, "jdbc:hive2://localhost:10000", "The URL for HiveServer2.")
```

Comment:
Is there a convention for the name of the property? It seems most commonly they are in the form of my.property.name